mistral/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:trusty
```

Comment:
Absolutely. I've noticed we're using it in st2 today, but it somehow slipped my mind that I'm depending on ubuntu in mistral containers.